Cargo.toml

```
@@ -50,7 +62,7 @@ rgb-schemata = "=0.10.1"
 rgb-std = "=0.10.9"
 rgb-wallet = "=0.10.9"
 rgb_core = { package = "rgb-core", version = "=0.10.8" }
-strict_encoding = "=2.6.1"
+strict_encoding = "2.6.2"
```

```suggestion
strict_encoding = "=2.6.2"
```
We prefer to keep this dependency version locked